### PR TITLE
docs: improvements

### PR DESCRIPTION
**Pull request by deepankarm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,25 +51,25 @@ Dive into understanding `fastapi-serve` through our comprehensive documentation
 - 🚀 **Getting Started**
     - 🧱 [Deploy a Simple FastAPI Application](docs/simple/)
     - 🖥️ [Dig deep into the `fastapi-serve` CLI](docs/CLI.md)
-    - ⚙️ [Understanding Configuration and Pricing](docs/CONFIG.MD)
+    - ⚙️ [Understanding Configuration and Pricing on Jina AI Cloud](docs/CONFIG.MD)
     - 🔄 [Upgrade your FastAPI applications with zero downtime](docs/upgrades/)
-    - 🏢 Managing Larger Applications with Complex Directory Structure (Example TBD!)
+    - 🏢 Managing Larger Applications with Complex Directory Structure (Coming Soon!)
```
Comment (deepankarm) [inline README.md]: I would like to separate `Coming Soon` & `Example TBD` as the former denotes a new feature whereas the latter is for pending docs for existing features.

Comment (scott-martens) [reply README.md]: "Example TBD" is not going to mean much to readers. How about if I say "Documentation in progress"?

Comment (deepankarm) [reply README.md]: Sounds good

Comment (scott-martens) [reply README.md]: Done.

**Review by scott-martens — APPROVED** [review]
I'm done. But maybe at least one more person should look it over?

**Pull request merged**